src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,224 @@
+})<StyledProfileCardProps>(({ theme, $variant }) => {
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(0.5),
+    textAlign: variantConfig.textAlign,
+    flex: $variant === 'compact' ? 1 : 'initial',
+    minWidth: 0,
+    [theme.breakpoints.down('sm')]: {
+      textAlign: 'center',
+    },
+  };
+});
+
+export const StyledName = styled(Typography, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const typographyConfig = PROFILE_CARD_TYPOGRAPHY[$size];
+
+  return {
+    fontSize: typographyConfig.nameSize,
+    fontWeight: theme.typography.fontWeightBold || 600,
+    color: theme.palette.text.primary,
+    lineHeight: theme.typography.body1.lineHeight,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.body1.fontSize,
+    },
+  };
+});
+
+export const StyledUsername = styled(Typography, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const typographyConfig = PROFILE_CARD_TYPOGRAPHY[$size];
+
+  return {
+    fontSize: typographyConfig.usernameSize,
+    fontWeight: theme.typography.fontWeightRegular || 400,
+    color: theme.palette.grey[600],
+    lineHeight: theme.typography.body2.lineHeight,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.body2.fontSize,
+    },
+  };
+});
+
+export const StyledRole = styled(Typography, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const typographyConfig = PROFILE_CARD_TYPOGRAPHY[$size];
+
+  return {
+    fontSize: typographyConfig.roleSize,
+    fontWeight: theme.typography.fontWeightMedium || 500,
+    color: theme.palette.primary.main,
+    lineHeight: theme.typography.caption.lineHeight,
+    marginTop: theme.spacing(0.5),
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.caption.fontSize,
+    },
+  };
+});
+
+export const StyledStatsContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size, $variant }) => {
+  const spacingConfig = PROFILE_CARD_SPACING[$size];
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: variantConfig.statsDirection,
+    gap: theme.spacing(spacingConfig.statsGap),
+    marginTop: theme.spacing(spacingConfig.statsPaddingTop),
+    paddingTop: theme.spacing(spacingConfig.statsPaddingTop),
+    borderTop: `1px solid ${theme.palette.grey[200]}`,
```

Comment:
The border top is created using a hardcoded `1px` value: `borderTop: `1px solid ${theme.palette.grey[200]}``. Prefer theme tokens.

**Sugerencia:**
```typescript
Use a theme token for the border and spacing. For example: `borderTop: `${theme.palette.divider}`` or use `borderTopWidth: theme.spacing(0.125)` together with a palette color. Keep border thickness and color driven by the theme.
```